src/components/ErrorMessage/ErrorMessages.tsx

```diff
@@ -66,7 +66,7 @@ export class TransactionErrorMessage extends BaseErrorMessage {
         super(fallback);
         this.icon = <SentimentVeryDissatisfiedIcon />;
         this.text = text;
-        this.btnText = 'Start over';
+        this.btnText = 'Try again';
```

Comment:
As this is an error, maybe just for more declaration it can be: Error. Try Again

Otherwise the user doesn't explcility know that something went wrong